code/src/OSFramework/Enum/ErrorCodes.ts

```diff
@@ -21,6 +21,9 @@ namespace OSFramework.Enum {
         API_FailedApplyRowValidation = 'GRID-API-02005',
         // VIEW
         API_FailedGetViewLayout = 'GRID-API-03001',
-        API_FailedSetViewLayout = 'GRID-API-03002'
+        API_FailedSetViewLayout = 'GRID-API-03002',
+        //SORT
+        API_FailedClear = 'GRID-API-04001',
+        API_FailedColumnSort = 'GRID-API-04002'
```

Comment:
We might need to change this after merge, as on my side I also created codes for Selected API using 04...

Reply:
Hum I might have missed that part :/

Maybe after the merges we can do another PR to review all the codes